packages/frontend/src/app/home/home.component.html

```diff
@@ -17,23 +17,49 @@
 <!--PRESENTATION TEXT-->
 <div id="presentation-text">
     <h2>Welcome to flaky.dev !</h2>
-    <h3>
-        <p>This is a platform that allows you to track every test in your repository.</p>
-        <p>If any of them is flaky, it will be found !</p>
-    </h3>
 </div>
 
-<!--INTRODUCTION TEXT-->
-<div id="introduction-text">
-    <p>What are you going to do now?</p>
-</div>
 
-<!--SEARCH BAR-->
-<div id="search-bar">
-    <app-search (searchOptionSelected)="onSearchOptionSelected($event)"></app-search>
+<!--DESCRIPTIONS-->
+<div class="description">
+
+    <section class="description-section">
+        <h3 class="description-header">One place for every test</h3>
+        <p class="description-text">
+            Track every test in your repositories.
+            <br/>
+            Automatically open new issues for failling tests.
+        </p>
+    </section>
+
+    <section class="description-section">
+        <h3 class="description-header">Spot flaky tests</h3>
+        <p class="description-text">
+            Find out which tests pass and fail nondeterministically.
```

Comment:
Based on historical build information, flaky.dev calculates which of your tests exhibit flakiness.